### DISCUSSION TITLE
Why no deb packages?

### DISCUSSION THREAD

**Discussion by mkczyk** [transmission/transmission#6455] [Q&A] [answered]
There is no `transmission-daemon` deb package for newer/older version in apt.

For example for Raspberry Pi (Raspbian 11 Bullseye) there is only `3.00-1` version (which is [bugged](https://github.com/transmission/transmission/issues/1314)). You can check by `apt-cache policy transmission-daemon`.

There are no version 4.x or newer 3.x or even old 2.x.

Installing from sources (you have to install dependencies and you can't uninstall easily) or use packages from Ubuntu aren't so easy (or just doesn't work) as just:
```
sudo apt install transmission-daemon=4.0.5
```

**Comment by nevack** [comment, marked as answer]
I think it is up to Raspbian maintainers to build a newer version.

Transmission developers do not influence this.
